src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddBagToDeposit.scala

```diff
@@ -17,31 +17,34 @@ package nl.knaw.dans.easy.multideposit.actions
 
 import java.io.{ File, FileInputStream }
 
-import gov.loc.repository.bagit.BagFactory
 import gov.loc.repository.bagit.BagFactory.Version
 import gov.loc.repository.bagit.Manifest.Algorithm
-import gov.loc.repository.bagit.transformer.impl.DefaultCompleter
+import gov.loc.repository.bagit.transformer.Completer
+import gov.loc.repository.bagit.transformer.impl.{ ChainingCompleter, DefaultCompleter, TagManifestCompleter }
 import gov.loc.repository.bagit.utilities.MessageDigestHelper
 import gov.loc.repository.bagit.writer.impl.FileSystemWriter
+import gov.loc.repository.bagit.{ Bag, BagFactory }
 import nl.knaw.dans.easy.multideposit._
 import nl.knaw.dans.easy.multideposit.actions.AddBagToDeposit._
+import org.joda.time.DateTime
+import org.joda.time.format.ISODateTimeFormat
 
-import scala.collection.JavaConversions.seqAsJavaList
+import scala.collection.JavaConverters._
 import scala.util.control.NonFatal
 import scala.util.{ Failure, Try }
 
-case class AddBagToDeposit(row: Int, datasetID: DatasetID)(implicit settings: Settings) extends Action {
+case class AddBagToDeposit(row: Int, datasetID: DatasetID, dataset: Dataset)(implicit settings: Settings) extends Action {
```

Comment:
I liked the entry: (DatasetID, Dataset), which is used by the other Actions. 
Could you use it here too?